### DISCUSSION TITLE
Livarno HG06106C - some working some can't change colour via Z2M

### DISCUSSION THREAD

**Discussion by jamesd256** [Koenkk/zigbee2mqtt#6859] [Devices]
Hi,

Working HG06106C - everything is as normal and expected

Non working HG06106C  - The bulb is physically capable of colour, because if you select a colour, while nothing happens immediately, if you then select any of the effects, the colour choice is immediately reflectded.

Cannot see anything to differentiate the two in terms of properties reported by Z2M (about and exposes tabs on web interface)

The payloads and states look the same in Z2M web i/f and MQTTExplorer

Thanks!

James

**Comment by jamesd256** [comment]
After buying a few more of these, it turns out there are some distinct versions of the HG06106C.   If you look at the cluster of logos on the light, the latest ones have a small cluster constrained to a square area, whereas the ones that don't work have a slightly larger cluster and one logo sticks out to the right.  These second ones are the ones that don't respond to colour change correctly.  Luckily I had 4 of the 16 I bought and these could be returned

**Comment by py-g** [comment]
> Hi,
> 
> Working HG06106C - everything is as normal and expected
> 
> Non working HG06106C - The bulb is physically capable of colour, because if you select a colour, while nothing happens immediately, if you then select any of the effects, the colour choice is immediately reflectded.
> 
> Cannot see anything to differentiate the two in terms of properties reported by Z2M (about and exposes tabs on web interface)
> 
> The payloads and states look the same in Z2M web i/f and MQTTExplorer
> 
> Thanks!
> 
> James

Hello James and others
I had the exact same problem with some of my HG06106Cs.
Physically unplugging them and then plugging them back in solved this problem.
I spent hours in the logs and code trying to figure it out and the solution was finally that simple...
